### PR TITLE
Feature/error handler

### PR DESCRIPTION
**Pull request by marioncottard**

### CODE REVIEW

File: cornflow-server/cornflow/tests/ldap/test_ldap_authentication.py [hunk 1]
```diff
@@ -116,4 +116,7 @@ def test_deactivated_sign_up(self):
             follow_redirects=True,
             headers={"Content-Type": "application/json"},
         )
+        print(response)
+        print(response.__dict__)
+        print(response.json)
```
Comment (ggsdc) [inline cornflow-server/cornflow/tests/ldap/test_ldap_authentication.py:121]: This should be left out

File: libs/core/cornflow_core/exceptions/exceptions.py [hunk 2]
```diff
@@ -4,7 +4,9 @@
 """
 from flask import jsonify
 from webargs.flaskparser import parser
+from werkzeug.exceptions import InternalServerError
```
Comment (ggsdc) [inline libs/core/cornflow_core/exceptions/exceptions.py:7]: `InternalServerError` is not used anywhere?

**Review by ggsdc — COMMENTED** [review]
Is this PR finished?

File: cornflow-server/cornflow/tests/ldap/test_ldap_authentication.py (hunk 1 above)
Comment (marioncottard) [reply cornflow-server/cornflow/tests/ldap/test_ldap_authentication.py]: Done

File: libs/core/cornflow_core/exceptions/exceptions.py (hunk 2 above)
Comment (marioncottard) [reply libs/core/cornflow_core/exceptions/exceptions.py]: You're right

**Pull request merged**